src/ddspdrum/torchmodule.py

```diff
@@ -485,3 +485,146 @@ def _forward(self, control_in: T, audio_in: T) -> T:
 
         audio_in = fix_length(audio_in, len(control_in))
         return control_in * audio_in
+
+
+class TorchFIR(TorchSynthModule):
```

Comment:
Considering renaming this -- FIRLowPass or something similar that is more specific